### DISCUSSION TITLE
Question about the amps

### DISCUSSION THREAD

**Discussion by Larswagner1987** [danielnilsson9/bbs-fw#258] [General]
Hello, why do I only get 1400 watts when the setting is 32 amps?? I have the problem that when I have a bike trailer attached I only get a maximum of 1000 watts. The trailer weighs a maximum of 15 kilograms with shopping. I don't know why this is. The motor has been serviced because it didn't have enough oil. Does anyone have a tip on how I can better climb a gradient of up to 15 percent when I have the trailer attached? It's really slow when I'm riding up the hill at 7-9 km/h and I have to use a lot of energy. Thanks in advance.

**Comment by silenec** [comment]
Use your gears, not amps. I can ride up any hill with a 30kg trailer using less than 700 watts even without pedaling. I could even go up the hill without motor assistance. Not very fast of course but I am not racing with a trailer. I could go faster with more power but that would stress the drivetrain more and also the motor + battery temps would go up. Going slow while pedaling fast is the way.

**Reply by Larswagner1987** [reply]
Well, if my motor is off then it is difficult for me to drive up a hill or a slight incline under my own power. I turn like crazy when I have the trailer attached, but what could be causing it to be so difficult? I have now set it to 32 A and now I can drive up the hill with the trailer at around 18 km/h but the battery is really suffering. Has something jammed on the motor or what could it be? My wife does not have the problem, the motor accelerates really well, but not for me and we have the same settings on the motor and the same on the bike. I am grateful for any tips.
